server/routes/todo.js

```diff
@@ -26,10 +28,10 @@ router.put('/', function(req, res, next) {
 
 
 /* DELETE todo list entry*/
-router.delete('/', function(req, res, next) {
-    // TODO implementation deleting entry from our database
-
-    res.send('delete todo list entry'); // TODO change response after
+router.delete('/:userId', function(req, res, next) {
+    // // TODO Error handling
+    database.ref(req.params.userId).remove();
```

Comment:
This is for deleting a user, the issue is for deleting an todolist item for a user

Reply:
Ah, I saw the firebase was empty yesterday. I thought it might using different tables to store users information and to-do list. If they are stored together, I do need to change this one. Thank you~